yajsapi/utils/promisify.ts

```diff
@@ -1,4 +1,4 @@
-export default function promisify(fn: Function): Function {
+export default function promisify(fn: (...args) => void): (arg) => Promise<any> {
```

Comment:
I think I don't get the type definition for `fn` here -> doesn't it limit the functions passed here to ones that don't return any values? is that what is wanted?

Reply:
yes, it should be `(...args) => any`